src/simplnx/Utilities/DataGroupUtilities.cpp

```diff
@@ -97,7 +98,7 @@ bool RemoveInactiveObjects(DataStructure& dataStructure, const DataPath& feature
           std::vector<DataPath> neighborListDataPaths = result.value();
           for(const auto& neighborListDataPath : neighborListDataPaths)
           {
-            dataStructure.removeData(neighborListDataPath);
+            messageHandler(nx::core::IFilter::Message{nx::core::IFilter::Message::Type::Info, fmt::format("NeighborList '{}' will be from the DataStructure.", neighborListDataPath.toString())});
```

Comment:
- This message doesn't make sense, there is no action verb.

- Also, the changelog shows there used to be a removal here that was replaced by the method, and I don't see any additions to this file that would actually remove the data. If this was intentional, I would rename the method to make clear it won't actually remove anything or add a removal function.

Reply:
The removal will happen via a deferred DeleteDataAction. I am sending an information message to the user so it prints in the output log. They should have already received a warning about the NeighborLists being removed.


I will fix the sentence structure.

Reply:
messageHandler(nx::core::IFilter::Message{nx::core::IFilter::Message::Type::Info, fmt::format("NeighborList '{}' will be removed from the DataStructure.", neighborListDataPath.toString())});
```suggestion
            messageHandler(nx::core::IFilter::Message{nx::core::IFilter::Message::Type::Info, fmt::format("NeighborList '{}' will be from the DataStructure.", neighborListDataPath.toString())});
```